README.md

```diff
@@ -56,20 +60,21 @@ Purgers do all the hard work of telling external systems what to invalidate
 and do this in the technically required way, for instance with external API
 calls, through telnet commands or with specially crafted HTTP requests.
 
-Purge **doesn't ship any** purger, as this is context specific. You could for
+Purge **doesn't ship any purger**, as this is context specific. You could for
 instance have multiple purgers enabled to both clean a local proxy and a CDN
 at the same time.
 
 #### Processing Policies
 Although editing content leads to ``tag`` purgeables automatically getting
 queued, this doesn't mean they get processed automatically. It is up to you
-to select configure a stable situation for your needs.
+to select a stable configuration for your needs.
 
 Policy possibilities:
 
-* **none** tags get queued, but nothing clears automatically.
-* **``cron``** claims from the queue & purges during cron.
-* **``ajaxui``** after editing content, an AJAX-based progressbar works the queue.
+* **``none``** tags get queued, but nothing gets cleared automatically.
```

Comment:
No, please remove the backticks, because there's no plugin/submodule named 'none' at all, it was deliberately only bold because simply means 'no checkboxes checked'.